### PR TITLE
v4 spelling fixes

### PR DESCRIPTION
**Pull request by diberry**

### CODE REVIEW

File: csharp_dotnetcore/13.Basic-Bot-Template/Dialogs/Greeting/GreetingDialog.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ namespace Microsoft.BotBuilderSamples
     /// <summary>
     /// Demonstrates the following concepts:
     /// - Use a subclass of ComponentDialog to implement a mult-turn conversation
-    /// - Use a Waterflow dialog to model multi-turn conversation flow
+    /// - Use a Waterfall dialog to model multi-turn conversation flow
```
Comment (stevengum) [inline csharp_dotnetcore/13.Basic-Bot-Template/Dialogs/Greeting/GreetingDialog.cs]: For `"Waterfall dialog"` it should be `"WaterfallDialog"`, can you change this line to that and for all the other lines as well?

File: javascript_nodejs/13.basic-bot/bot.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ class Bot {
         if (!userState) throw ('Missing parameter.  userState is required');
         if (!botConfig) throw ('Missing parameter.  botConfig is required');
 
-        // add the LUIS recogizer
+        // add the LUIS recognizer
```
Comment (stevengum) [inline javascript_nodejs/13.basic-bot/bot.js]: Should be `"Add the LuisRecognizer."`

File: javascript_nodejs/14.nlp-with-dispatch/dialogs/homeAutomation/index.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ class homeAutomation {
         // home automation state
         this.state = new HomeAutomationState(convoState, userState);
         
-        // add recogizers
+        // add recognizers
```
Comment (stevengum) [inline javascript_nodejs/14.nlp-with-dispatch/dialogs/homeAutomation/index.js]: Should also be `"Add the LuisRecognizer."` for this line and the other changes below, unless there are actually multiple recognizers being registered.

**Comment by cleemullins** [comment]
@stevengum @diberry These changes look great, but there area  number of merge conflicts that need to be resolved prior to merging them in.

**Comment by cleemullins** [comment]
Closing this out due to an overwhelming number of merge conflicts. 

@diberry I would love to take these changes, if you're able to update the PR.

**Pull request closed without merging**